docs/specification/current.md

```diff
@@ -929,6 +917,20 @@ As another example, an extension defined on the discovery response could look li
 ```
 [OAuth 2.0]: https://oauth.net/2/
 
+## Data Types
+
+This section defines some data types that are used across the specification. These types are generic enough so that they may be used with various properties pertaining to different concepts in the text of the specification.
```

Comment:
```suggestion
This section defines data types used re-used across the specification. 
```

Reply:
You have a repeated word in your suggestion.

Reply:
I'm accepting this, and then I'll fix the repetition